libs/sdk-ui-vis-commons/src/utils/calculateCustomHeight.ts

```diff
@@ -57,3 +68,16 @@ export function calculateHeadlineHeightFontSize(
 
     return { height, fontSize };
 }
+
+/**
+ * @public
```

Comment:
This should be `@internal` since it is in the `vis-commons` package and only for internal use.